e2e/features/networkManagement/NetworkManagement.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Network Management
+  Background: Add an account and navigate open Switch network modal
+    Given I go to page "wallet"
+    And I wait for "1 seconds"
+    And I click on a button with text "customNode"
+    Then "customNode" should be selected in "Switch network" dropdown
+    Given I click on text "Add network"
+    Then I should be redirected to route: 'wallet?modal=dialogAddNetwork'
```

Comment:
I feel this should be "Then I should be redirected to addNetwork modal" or something similar. That way other modal views will be written in the same manner and the step definition can use the routes file

Reply:
Fixed https://github.com/LiskHQ/lisk-desktop/pull/5345/commits/48b2b682db531c4a62f09551ec22c8e497f293cd